src/web3-utils.js

```diff
@@ -106,5 +106,9 @@ export function getUnknownBalance() {
   return new BN('-1')
 }
 
+export function isValidEnsName(name) {
+  return /^([a-z0-9]+\.)+eth$/.test(name)
```

Comment:
Is there a length limit for `ENS` names?

Reply:
Only for the first level after `eth`, but I wouldn't worry too much about it (the ENS resolver would reject it anyway) and it's subject to change in the future.

Reply:
There is not a length limit on all ENS names.  For instance .luxe does not have any.